stopWatch.js

```diff
@@ -0,0 +1,248 @@
+
```

Comment:
general comment. 
Since you know type of some of the things like secs, mins, hrs, You can enforce the types using typescript as well. That would make it more usable.